src/app.controller.ts

```diff
@@ -12,7 +12,7 @@ export class AppController {
 
   @Get()
   get() {
-    console.log('hello there')
+    console.log('hello there ?')
```

Comment:
PullProofAI

The log message seems to have a question mark added unnecessarily. Consider removing it.
```diff
-     console.log('hello there ?')
+     console.log('hello there')
```


<!-- This is an auto-generated comment by PullProofAI -->